docs/computing.md

```diff
@@ -0,0 +1,20 @@
+# Efficient cluster Computing patterns
+
+`Nota bene`: The cluster can accomodate so much IO at any one point. IO is where computing breaks. At the node level and certainly at the cluster level.
```

Comment:
Not sure whether our average reader understands that "so much" doesn't actually mean "a lot of". Maybe we can just rephrase as "a finite amount of".